docs/src/style.md

```diff
@@ -179,6 +179,19 @@ some_local_variable = ...
 some_file.jl # Except for ModuleName.jl.
 ```
 
+Begin private functions and constants with an underscore. All other objects
+in the scope of a module should be exported.
+```julia
+function _private_jump_function end
+function public_jump_function end
+```
+
+Never begin a variable name with an underscore. We make an exception for
```

Comment:
Another way to say this is that starting underscores should be used only for distinguishing between exported and non-exported names. Therefore don't use them for local variables.